tomviz/FileFormatManager.h

```diff
@@ -4,6 +4,8 @@
 #ifndef tomvizFileFormatManager_h
 #define tomvizFileFormatManager_h
 
+#include <mutex>
```

Comment:
We may want to consider the Qt mutex classes which have read locks (multiple readers) and write locks (only one write lock, no read locks). I suspect we aren't that multithreaded, but worth considering. In pure Qt code there are often advantages in just using the Qt classes for better integration, this may not matter too much here.